src/components/Identity/getEcidCommand/createIdentityPayload.js

```diff
@@ -0,0 +1,18 @@
+import createRequestPayload from "../../../core/edgeNetwork/requestPayloads/createRequestPayload";
+
+export default () => {
+  return createRequestPayload(content => {
+    return {
+      addIdentity: (namespaceCode, identity) => {
+        content.identityMap = content.identityMap || {};
+        content.identityMap[namespaceCode] =
+          content.identityMap[namespaceCode] || [];
+        content.identityMap[namespaceCode].push(identity);
+      },
+      addQuery: () => {
+        content.query = content.query || {};
+        content.query.identity = { fetch: ["ECID"] };
+      }
```

Comment:
If the query to fetch the identity is always required for this type of request, which I believe it is, then I think it would be best to avoid ever being able to create a request payload without the query. In other words, remove `addQuery` from here and enforce that `content.query.identity` is always set by just moving this code:

```
content.query = content.query || {};
content.query.identity = { fetch: ["ECID"] };
```
to before [this line](https://github.com/adobe/alloy/pull/385/files#diff-2b6fe5249e78819450208158631fd544R5).